### ISSUE TITLE
Uninstalling causes error for all apps

### ISSUE THREAD

**Issue by Ghasan** [ScoopInstaller/Scoop#1932]
I noticed today that when I install any app, I get three error messages and then it uninstalls fine. I tried multiple times with re-installing the apps and uninstallaing them again.

I checked my 'git status' of scoop folder, and it is intact.

Here is an example:

> Uninstalling 'nano' (2.5.3).
Removing shim for 'nano'.
Unlinking ~\scoop\apps\nano\current
The system cannot find the file specified.
The system cannot find the file specified.
The system cannot find the path specified.
'nano' was uninstalled.

**Comment by dsbert** [comment]
Did you recently install the creators update? I noticed some strange behavior after that which caused me to have to reset some of the scoop installs.

**Comment by Ghasan** [comment]
I tried to do some debugging, and arrived at the culprit line.

https://github.com/lukesampson/scoop/blob/590fb80cb40c3a49714320b3f56d42046f185bb7/lib/install.ps1#L771

` & "$env:COMSPEC" /c "rmdir $currentdir"`

I tried to execute the code manually from PS, but I get Access Denied! Even though the folder is owned by the current users, and has full access to the folder.

However, when done using scope it successfully deletes the folder, but still prints those weird 'cannot find' errors.
